src/helpers/getFormattingStatus.ts

```diff
@@ -8,9 +8,16 @@ const getFormattingStatus = () => {
     DEFAULT_AFFECTS_CONFIGURATION,
   )
   const isAnyRelevantSettingActivated = affectsConfiguration.some(setting => {
-    const isSettingActivated = configuration.get(setting, false)
+    if(!setting.includes('editor.codeActionsOnSave')) {
+      const isSettingActivated = configuration.get(setting, false)
 
-    return isSettingActivated
+      return isSettingActivated
+    }
+
+    const codeActionsOnSave: Object = configuration.get('editor.codeActionsOnSave', {});
```

Comment:
Since the `vscode` package is typed, TypeScript would automatically infer what the `configuration.get` method returns. In this case it would type it as the type of the default value, which is an object literal. So you don’t need to manually add the `: Object` type. :)

<img width="655" alt="Screenshot 2022-07-17 at 20 04 16" src="https://user-images.githubusercontent.com/9154236/179418922-d9eecbf7-2537-4ba7-9926-29b4b0459cad.png">

---

However I rewrote your commit and did I things a little differently. I think it’s a little nicer by using `workspace.getConfiguration('editor.codeActionsOnSave')`!